website/docs/d/networkmanager_core_network_policy_document.html.markdown

```diff
@@ -0,0 +1,255 @@
+* `conditions` - A block argument. Detailed Below.
+* `description` (Optional) - A user-defined description that further helps identify the rule.
+* `rule_number` - An integer from `1` to `65535` indicating the rule's order number. Rules are processed in order from the lowest numbered rule to the highest. Rules stop processing when a rule is matched. It's important to make sure that you number your rules in the exact order that you want them processed.
+
+### `action`
+
+The following arguments are available:
+
+* `association_method` - Defines how a segment is mapped. Values can be `constant` or `tag`. `constant` statically defines the segment to associate the attachment to. `tag` uses the value of a tag to dynamically try to map to a segment.reference_policies_elements_condition_operators.html) to evaluate.
+* `segment` - The name of the `segment` to share as defined in the `segments` section. This is used only when the `association_method` is `constant`.
+* `tag_value_of_key` - Maps the attachment to the value of a known key. This is used with the `association_method` is `tag`. For example a `tag` of `stage = “test”`, will map to a segment named `test`. The value must exactly match the name of a segment. This allows you to have many segments, but use only a single rule without having to define multiple nearly identical conditions. This prevents creating many similar conditions that all use the same keys to map to segments.
+* `require_acceptance` - Determines if this mapping should override the segment value for `require_attachment_acceptance`. You can only set this to `true`, indicating that this setting applies only to segments that have `require_attachment_acceptance` set to `false`. If the segment already has the default `require_attachment_acceptance`, you can set this to inherit segment’s acceptance value.
+
+### `conditions`
+
+The conditions block has 4 arguments `type`, `operator`, `key`, `value`. Setting or omitting each argument requires a combination of logic based on the value set to `type`. For that reason, please refer to the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/cloudwan/cloudwan-policies-json.html) for complete usage docs.
+
+The following arguments are available:
+
+* `type` - Valid values include: `account-id`, `any`, `tag-value`, `tag-exists`, `resource-id`, `region`, `attachment-type`.
+* `operator` - Valid values include: `equals`, `not-equals`, `contains`, `begins-with`.
+* `key` - string value
+* `value` - string value
+
+### `core_network_configuration`
+
+The following arguments are available:
+
+* `asn_ranges` - List of strings containing Autonomous System Numbers (ASNs) to assign to Core Network Edges. By default, the core network automatically assigns an ASN for each Core Network Edge but you can optionally define the ASN in the edge-locations for each Region. The ASN uses an array of integer ranges only from 64512 to 65534 and 4200000000 to 4294967294. No other ASN ranges can be used.
```

Comment:
We can change "64512" to "`64512`" for readability. Same for the other numbers for this argument, to be consistent with how other parts of the documentation have them highlighted